apps/web/public/.well_known/walletconnect.txt

```diff
@@ -0,0 +1 @@
+074c7530-876a-4af0-98ca-b084e3586fb5=32eb4d878d84c69d3d9d3aaf789e24dd0a2317a861e255e95268dc3dab06fb75
```

Comment:
what is this?

Reply:
It's for wallet connect domain verification